lib/parsers/simple.js

```diff
@@ -14,21 +14,23 @@ module.exports = function($) {
     if (!$) return resolve({});
     var result = {};
 
+    var value;
+
     // title
     var title = $('title');
     if (title) result.title = title.text().trim();
 
     // description
     var description = $('meta[name="description"]');
     if (description) {
-      var value = description.attr('content');
+      value = description.attr('content');
       if (value) result.description = value.trim();
     }
 
     // theme color
     var themeColor = $('meta[name="theme-color"]');
     if (themeColor) {
-      var value = themeColor.attr('content');
+      value = themeColor.attr('content');
```

Comment:
Not sure why it didn't like `let` or `const` block scopings here. So I moved the declaration up one level.